docs/guide/deployment/http-gateway.md

```diff
@@ -0,0 +1,225 @@
+# HTTP gateway
+Http gateway exposes REST API for devices, cloud configuration, certificate authority and websockets for events as well as user web UI (i.e. PLGD Dashboard).
```

Comment:
HTTP Gateway exposes the client's [REST API](swagger...) to manage the user's devices, as well as the Web UI known as [plgd Dashboard](... for now leave three dots, will update later).